front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -10,21 +11,33 @@ import type { PathfindingResultSuccess, TrainScheduleResult } from 'common/api/o
 import { formatSuggestedOperationalPoints } from 'modules/pathfinding/utils';
 import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainSchedule/types';
 import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+import { convertIsoUtcToLocalTime } from 'utils/date';
 import { secToHoursString } from 'utils/timeManipulation';
 
 import { checkAndFormatCalculatedArrival } from '../helpers/arrivalTime';
 import computeMargins from '../helpers/computeMargins';
 import { computeScheduleData, formatScheduleData } from '../helpers/scheduleData';
-import { findNextScheduledOpPoint } from '../helpers/utils';
-import type { TrainScheduleBasePathWithUic, ScheduleEntry } from '../types';
+import {
+  findNextScheduledOpPoint,
+  formatSuggestedViasToRowVias,
+  updateDaySinceDeparture,
+} from '../helpers/utils';
+import {
+  type TrainScheduleBasePathWithUic,
+  type ScheduleEntry,
+  type TimeStopsRow,
+  TableType,
+} from '../types';
 
-function useOutputTableData(
+const useOutputTableData = (
```

Comment:
I don’t see the point.
arrow functions were made to simplify writing callbacks like `setTimeout(() => { doStuff() }, 2000)` or higher level function
() => {} were never meant to replace `function`

if you remove every single function and replace by `const` you completely lose the visual cues, you have no more distinction between const functions and const everything else.

Reply:
alright, added it to next front meeting